lib/elixir/test/elixir/string_test.exs

```diff
@@ -377,26 +389,112 @@ defmodule StringTest do
     assert String.duplicate("abc", 1) == "abc"
     assert String.duplicate("abc", 2) == "abcabc"
     assert String.duplicate("&ã$", 2) == "&ã$&ã$"
+
     assert_raise ArgumentError, fn ->
       String.duplicate("abc", -1)
     end
   end
 
   test "codepoints/1" do
     assert String.codepoints("elixir") == ["e", "l", "i", "x", "i", "r"]
-    assert String.codepoints("elixír") == ["e", "l", "i", "x", "í", "r"] # slovak
-    assert String.codepoints("ոգելից ըմպելիք") == ["ո", "գ", "ե", "լ", "ի", "ց", " ", "ը", "մ", "պ", "ե", "լ", "ի", "ք"] # armenian
-    assert String.codepoints("эліксір") == ["э", "л", "і", "к", "с", "і", "р"] # belarussian
-    assert String.codepoints("ελιξήριο") == ["ε", "λ", "ι", "ξ", "ή", "ρ", "ι", "ο"] # greek
-    assert String.codepoints("סם חיים") == ["ס", "ם", " ", "ח", "י", "י", "ם"] # hebraic
-    assert String.codepoints("अमृत") == ["अ", "म", "ृ", "त"] # hindi
-    assert String.codepoints("স্পর্শমণি") == ["স", "্", "প", "র", "্", "শ", "ম", "ণ", "ি"] # bengali
-    assert String.codepoints("સર્વશ્રેષ્ઠ ઇલાજ") == ["સ", "ર", "્", "વ", "શ", "્", "ર", "ે", "ષ", "્", "ઠ", " ", "ઇ", "લ", "ા", "જ"] # gujarati
-    assert String.codepoints("世界中の一番") == ["世", "界", "中", "の", "一", "番"] # japanese
+    # slovak
+    assert String.codepoints("elixír") == ["e", "l", "i", "x", "í", "r"]
+    # armenian
+    assert String.codepoints("ոգելից ըմպելիք") == [
+             "ո",
+             "գ",
+             "ե",
+             "լ",
+             "ի",
+             "ց",
+             " ",
+             "ը",
+             "մ",
+             "պ",
+             "ե",
+             "լ",
+             "ի",
+             "ք"
+           ]
```

Comment:
Please break those large lists into smaller lists with list concatenation. For example:

```elixir
assert String.codepoints("ոգելից ըմպելիք") ==
         ["ո", "գ", "ե", "լ", "ի", "ց", " "] ++ ["ը", "մ", "պ", "ե", "լ", "ի", "ք"]